workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -63,6 +63,7 @@ class UserConfig:
     online_emulator: Optional[
         Union[runtime.transformers.emulator.Config, runtime.transformers.fv3fit.Config]
     ] = None
+    scale_tendencies: Mapping[str, float] = dataclasses.field(default_factory=dict)
```

Comment:
Feels odd for this to be a top-level config item. Why not add it as part of `MachineLearningConfig` since that is where it is intended to be used (if I understand correctly).